CHANGELOG-fix-protocol-links.md

```diff
@@ -0,0 +1,2 @@
+- Restore protocol links on donor pages.
```

Comment:
Are these entries duplicates of those in `CHANGELOG-hmp-152`?

Reply:
Yes, I'll remove this file; I think I preemptively added the hmp-152 changelog when I started this work, then forgot about it when I came back to finish it :)